.github/workflows/close-stale-issues.yaml

```diff
@@ -0,0 +1,23 @@
+name: Close stale issues and PRs
+
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+    stale:
+        name: Close stale issues and PRs
+        runs-on: ubuntu-latest
+        steps:
+        - name: Close stale issues and PRs
+          uses: actions/stale@v8
+          with:
+          stale-issue-message: 'Marking this issue as stale after 30 days of inactivity, please comment or this will be closed.'
```

Comment:
Here are some issues that document known deficiencies of lakeFS.  Of course we could close them every 30 days and open a new one, or add comments every 29 days.  But *why* do I need to work for the tooling?

Some of these have user requirements or discussions on them, it would be particularly painful to lose those.

* #1371 
* #1405
* #1605
* #1607
* #1726
* #1742 
* #1765
* #1804
* #1849 
* #1911 
* #1971 
* #2092 
* #2150
* #2228 
* #2308 
* #2491 
*